### PR TITLE
Pushing commit from v3.x to release/3.0

### PR DESCRIPTION
**Pull request by v-imohammad**
### Issue describing the changes in this PR

cherry-picked below list of commits from v3.x to release/3.0
Adding runtime site name to valid JWT audiences (slot scenarios). (#10186)
Defaulting SwtAuthenticationEnabled to False (#10196)
Regenerate .gitignore via dotnet cli (#10206)
[v3.x] Add 1ES CI pipelines (#10247)
Address test secrets being flagged (#10264)
Release Prep for 3.23.0 (#10291)

resolves #issue_for_this_pr

### Pull request checklist

**IMPORTANT**: Currently, changes must be backported to the `in-proc` branch to be included in Core Tools and non-Flex deployments.

* [ ] Backporting to the `in-proc` branch is not required
    * Otherwise: Link to backporting PR
* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

**Review by jviau — APPROVED** [review]

**Comment by arroyc** [comment]
all the commits are there ... so, commit wise looks good ... let's wait for the build to be successful and then we can merge this, if we can't merge this, we probably need to ask for help again

**Review by arroyc — APPROVED** [review]

**Pull request merged**
